src/python/antares_xpansion/config_loader.py

```diff
@@ -79,15 +81,22 @@ def _set_simulation_name(self):
             raise ConfigLoader.MissingSimulationName(
                 "Missing argument simulationName")
         elif self._config.simulation_name == "last":
-            self._simulation_name = self._config.simulation_name
+            self._xpansion_simulation_name = self._config.simulation_name
 
         else:
             tmp_zip = Path(self.antares_output()) / \
                 self._config.simulation_name
-            if self.is_zip(tmp_zip):
-                self._last_zip = tmp_zip
-                self._simulation_name = self._last_zip.parent / \
-                    (self._last_zip.stem+"-Xpansion")
+            if self.is_antares_study(tmp_zip):
+                self._last_study = tmp_zip
+                if self.step() in ["resume", "sensitivity"] :
```

Comment:
Lines 91 - 99 look very much like the code of `_set_options_for_benders_solver`, see comment below